tests/test_define.h

```diff
@@ -39,6 +39,10 @@ extern "C" {
 #define CURRENT_INDEX 1
 #define LAST_INDEX 2
 #define NONCE "THISTANGLEACCELERATORNONCES"
+#define MWM 14
+#define THRESHOLD 100
+#define NEIGHBOR_URI_1 "tcp://8.8.8.8:14265"
```

Comment:
Why did you choose these URL?

Reply:
`NEIGHBOR_URI_1` refers to [addneighbors exmple](https://docs.iota.org/docs/node-software/0.1/iri/references/api-reference#addneighbors) in IRI API reference, and `NEIGHBOR_URI_2` refers to [entangled/cclient/serialization/json/tests/shared.h](https://github.com/iotaledger/entangled/blob/70a9a1f780c5ba19437d6bcf1882f68ccddd7b0e/cclient/serialization/json/tests/shared.h#L56). If these addresses are not reasonable, please let me know and I will correct it as soon as possible.

Reply:
Take note of the links specified in `entangled`, as the form of comments here.

Reply:
Copy that!